src/posts/experimental-static-build.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Scaling Astro Sites to Thousands of Pages'
+description: 'A new experimental flag in Astro build enables building sites with tens of thousands of pages.'
+publishDate: 'January 25, 2022'
+authors:
+  - matthew
+lang: 'en'
+---
+
+Astro's about to get a lot faster with a newly built build system intended to scale to tens, or even hundreds, of thousands of pages. If you hang out in our [Discord](https://astro.build/chat) or pay attention to recently releases you might have seen a lot of discussion about a "static build". This is our new implementation of `astro build` that does 2 things:
```

Comment:
```suggestion
Astro's about to get a lot faster with a new build system intended to scale to tens, or even hundreds, of thousands of pages. If you hang out in our [Discord](https://astro.build/chat) or pay attention to recently releases you might have seen a lot of discussion about a "static build". This is our new implementation of `astro build` that does 2 things:
```

brevity? built build sounds weird to me :p

Reply:
Nit: `Astro's` => `Astro is`